localgov_elections_reporting.install

```diff
@@ -174,4 +174,27 @@ function localgov_elections_reporting_update_9003(&$sandbox){
   $charts_config = \Drupal::configFactory()->getEditable('charts.settings');
   $charts_config->set('advanced.requirements.cdn', TRUE);
   $charts_config->save();
-}
\ No newline at end of file
+}
+
```

Comment:
I'd add a comment here so it tells people what the update does. When you do `drush updb` the table will tell the user what the update is going to do.